scripts/test.sh

```diff
@@ -63,6 +63,10 @@ case $MODE in
     NODE_ENV=production webpack --config test/webpack.config.js --env.import-nothing --env.es6
     break;;
 
+  "browser-headless")
```

Comment:
`npm run test borwser-headless` already does this

Reply:
```
Running test suite in "borwser-headless" mode...
Unknown test mode borwser-headless

```